webview-ui/src/components/chat/checkpoints/CheckpointMenu.tsx

```diff
@@ -43,9 +45,11 @@ export const CheckpointMenu = ({ ts, commitHash, currentCheckpointHash }: Checkp
 
 	return (
 		<div className="flex flex-row gap-1">
-			<Button variant="ghost" size="icon" onClick={onCheckpointDiff} title="View Diff">
-				<span className="codicon codicon-diff-single" />
-			</Button>
+			{!checkpoint?.isFirst && (
```

Comment:
I don't think we want to expose the "diff" action for the initial checkpoint, but LMK if you feel differently.

Reply:
Agree